Sensus.Shared/SensusServiceHelper.cs

```diff
@@ -746,7 +755,10 @@ public void AddScriptToRun(Script script, RunMode runMode)
             }
 
             _scriptsToRun.Insert(0, script);
-            IssuePendingSurveysNotificationAsync(true, true);
+
+            var soundAndVibrate = ScriptIsWithinExclusionWindow(script);
+
+            IssuePendingSurveysNotificationAsync(soundAndVibrate, soundAndVibrate);
```

Comment:
I think these changes will be reverted once the alert-checking code is moved into the notifier.